src/Form.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+
+import Form from 'react-bootstrap/Form'
+import Button from 'react-bootstrap/Button'
+import axios from 'axios';
+import Card from 'react-bootstrap/Card'
+import Container from 'react-bootstrap/Container';
+
+
+
+class Forms extends React.Component {
+
+  constructor(props){
+    super(props);
+    this.state = {
+      city:"",
+      cityLat:"",
+      cityLon:"",
+      displayName:"",
+      cityMap:"",
+      errorCode:"",
+    }
+  }
+handleChange = (e) => {
+
+  this.setState({city: e.target.value})
+
```

Comment:
This function currently has no actual effect on the interface, as this.state.city is never used